runtime/ops/os.rs

```diff
@@ -151,7 +151,6 @@ fn op_loadavg(state: &mut OpState) -> Result<(f64, f64, f64), AnyError> {
 
 #[op]
 fn op_hostname(state: &mut OpState) -> Result<String, AnyError> {
-  super::check_unstable(state, "Deno.hostname");
   state.borrow_mut::<Permissions>().env.check_all()?;
```

Comment:
It seems sub-optimal that this requires granting permission to *all* environment variables. If some dependency needs to read the hostname, now I need to expose my entire env.

This issue also exists for these other unstable APIs.
- loadavg
- osRelease
- networkInterfaces
- systemMemoryInfo
- getGid
- getUid

Any thoughts on introducing a new permissions flag `--allow-sys-info` that accepts hard coded keys to expose each of these features individually?

Reply:
I like the idea. I created an independent issue for it https://github.com/denoland/deno/issues/15986